src/core/css/inc/mixins.scss

```diff
@@ -5,3 +5,15 @@
   box-shadow:0 0 5px 0 $focus-outline-color;
   outline:none;
 }
+
+@mixin respond-to($breakpoint) {
+  @if map-has-key($breakpoints, $breakpoint) {
+    @media #{map-get($breakpoints, $breakpoint)} {
+      @content;
+    }
+  }
```

Comment:
This generates quite a lot of separated media queries doesn't it? Or will they get merged on compilation to css? Not sure regarding your current guidelines but I agree it's sometimes nice to have all styles for one element in one place but it's also quite nice to see all styles related to a specific breakpoint in one place to reason about the changes and not search through the whole project.